lang/cs/Org.Apache.REEF.Network/Group/Driver/Impl/ContextManager.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I will move this to IMRU than here.....I suspect that later when we do FT we will need to make it IMRU specific